client/src/pages/dashboard/UserStore.ts

```diff
@@ -0,0 +1,9 @@
+import { asyncReadable } from '@square/svelte-store'
+import { Session } from '../../api/session.ts'
+
+export const userSession = asyncReadable(undefined, async() => {
+    const userData = await Session.getUser()
+    return userData;
+    }, 
+    { reloadable: true }
+);
```

Comment:
Prefer `null` over `undefined`.
```suggestion
export const userSession = asyncReadable(
    null,
    Session.getUser, 
    { reloadable: true }
);
```